.github/workflows/dev-deploy-db-migration.yaml

```diff
@@ -0,0 +1,38 @@
+name: Deploy DB Migration - Dev
+
+on:
+  push:
+    branches: [develop]
+  workflow_dispatch:
+
+# This will terminate builds that are previously, but continuing to run.  Saves GHA hours.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  helm-deployment:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Authenticate and set context
+        uses: redhat-actions/oc-login@v1.1
+        with:
+          openshift_server_url: ${{secrets.OPENSHIFT_CLUSTER_URL}}
+          openshift_token: ${{secrets.OPENSHIFT_PIPELINE_SERVICE_ACCOUNT_TOKEN}}
+
+      - name: deply db migration job
+        run: |
+          echo deleting dev-webapi-migrations job
```

Comment:
The test version of this file calls `oc project d8a8f9-test` here, do we need a `oc project d8a8f9-dev` in this one?